configure

```diff
@@ -165,6 +165,13 @@ echo "PKGCONFIGDIR ?= $PKGCONFIGDIR" >>$CONFIG_MK
 echo "TESTSDIR ?= $TESTSDIR" >>$CONFIG_MK
 echo "DBDIR ?= $DBDIR" >>$CONFIG_MK
 
+if [ -d .git ] && command -v git >/dev/null; then
```

Comment:
`$(git rev-parse --is-inside-work-tree 2>/dev/null) = true` instead of `-d .git` to handle worktrees?

Reply:
Ok!